### ISSUE TITLE
Bring Edit Layer Options to CBEFS Biological Prediction Layers

### ISSUE THREAD

**Issue by jamesdoyle21** [asascience/ott-coops#462]
The following 4 layers do not have any option besides opacity for edit layer. The client asked for a specific gradient for the pH layer, but instead of changing the defaults we can give the user the ability to change the gradient for all of these layers.

They are:
MARACOOS CBEFS 3D Aragonite Saturation

MARACOOS CBEFS 3D Oxygen

MARACOOS CBEFS 3D pH

MARACOOS CBEFS 3D DIC

![image](https://user-images.githubusercontent.com/82955395/167199273-f61f2d3e-2c48-4d42-854c-287216b95e20.png)

**Comment by nguyandy** [comment]
I don't think we support gradient selection for EDS(coastmap) layers on OM3 atm.

Here are some available gradients/style
<img width="138" alt="Screen Shot 2022-05-09 at 10 55 56 AM" src="https://user-images.githubusercontent.com/43226410/167437770-6af2596e-7e08-4889-bf17-0cfb3509fc50.png">
s available atm from EDS.

If the client wants a new custom color gradient, we will need help from the EDS team to define them.

**Comment by nguyandy** [comment]
@jamesdoyle21 yup! And after they have that custom gradient created, we'll need the style name from the EDS team

**Comment by mpiannucci** [comment]
Not only that, wed need the png to accompany it

**Comment by nguyandy** [comment]
Those 9 colors are the only ones that are available on OM1. (where we have the png to accompany it in the dropdown)

EDS does support more options, (ex: VIMS_OXYGEN_REVERSE from the other issue)
but we'll need the name/png as well like Matt mentioned if we want to display it in the layer options.

**Comment by jamesdoyle21** [comment]
@CoryMartinRPS

**Comment by CoryMartinRPS** [comment]
VIMS_OXYGEN_REVERSE is actually a pre-reversed version of the VIMS_OXYGEN style that already exists. I have added that to the oxygen layer, so you can now just do "styles=VIMS_OXYGEN" when requesting CHES_ROMS_HIGH_RES_OXYGEN and get the non-reversed version. If the style string is empty, it will default to the reversed.

Samples (copied from previous issue):
https://coastmap.com/ecop/wms.aspx?service=WMS&request=GetMap&version=1.1.1&layers=CHES_ROMS_HIGH_RES_OXYGEN&styles=VIMS_OXYGEN_REVERSE&width=426&height=706&format=image/png&transparent=true&time=2022-05-08T00:00:00.000Z&srs=EPSG:3857&bbox=-8613065.329826884,4377061.516340709,-8352655.415399621,4808640.685079252

https://coastmap.com/ecop/wms.aspx?service=WMS&request=GetMap&version=1.1.1&layers=CHES_ROMS_HIGH_RES_OXYGEN&styles=VIMS_OXYGEN&width=426&height=706&format=image/png&transparent=true&time=2022-05-08T00:00:00.000Z&srs=EPSG:3857&bbox=-8613065.329826884,4377061.516340709,-8352655.415399621,4808640.685079252

As for adding new colors, that would be a pretty simple addition to make. I recently refactored how the color maps are set up in the code, and made sure that it would be easy to add new ones if needed.

**Comment by jamesdoyle21** [comment]
@CoryMartinRPS I see the gradients coming up now for Oxygen. Do the other 3 layers have that option? MARACOOS CBEFS 3D Aragonite Saturation, MARACOOS CBEFS 3D pH, MARACOOS CBEFS 3D DIC?

And yes, if you could move forward with creating a gradient with blue denoting high values and red denoting low, that would be great.
